cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogDataOffsetCallback.java

```diff
@@ -50,4 +57,33 @@ public Object encodeSend(LogEvent logEvent) {
                                   ThrowableProxyUtil.asString(event.getThrowableProxy()));
     return new FormattedLogDataEvent(logData, logEvent.getOffset());
```

Comment:
Instead of returning `FormattedLogDataEvent` from here, if we directly return `modifyLogJsonElememnt(GSON.toJsonTree(new FormattedLogDataEvent(logData, logEvent.getOffset())))`, then we don't need to define a protected method `Object modifyLogJsonElememnt(JsonElement jsonElement)` in parent class, right?